### DISCUSSION TITLE
Anyone plz have v3.2.4??

### DISCUSSION THREAD

**Discussion by doinkmane** [utmapp/UTM#6213] [UTM Help (iOS)]
I’m running a jailbroken device on an older firmware which is mentioned supported by utm 3.2.4, but there is no download link for it on the releases page. can anyone who has the 3.2.4 IPA or deb help me get a copy? I’ve been looking everywhere to no avail…

**Comment by bewl87** [comment]
https://github.com/utmapp/UTM/releases/download/v3.2.4/UTM.deb
https://github.com/utmapp/UTM/releases/download/v3.2.4/UTM.ipa
